executable-spec/test/Cardano/Ledger/Spec/STS/Chain/Chain/Properties.hs

```diff
@@ -126,119 +171,102 @@ relevantCasesAreCovered
           "There are expired SIPs"
           $
 
-        -- X% of traces should: should have submitted SIPs
-        QC.cover 80
-          (submittedSIPsExist traceSample)
-          "Submitted SIPs exist"
-          $
-
-        -- X% of traces should: should have revealed SIPs
-        QC.cover 80
-          (revealedSIPsExist traceSample)
-          "Revealed SIPs exist"
-          $
-
-        -- X% of traces should: should have ballots for SIPs
-        QC.cover 80
-          (sipBallotsExist traceSample)
-          "Ballots for SIPs exist"
-          $
-
-        -- X% of traces should: should have voting results for SIPs
-        QC.cover 80
-          (voteResultsExist traceSample)
-          "Voting results for SIPs exist"
+          -- X% of traces should: There are no active SIPs with no votes
+        QC.cover 99
+          ( let SIPsVoteResults (vresmap) = Chain.vresips
```

Comment:
Hmmm, I'm thinking we should have a higher percentage of SIP's without votes. I'd expect this to happen in the actual system, albeit not that often.

Reply:
I have changed that